submissions/Mifaresss/task_js_oop/app.js

```diff
@@ -0,0 +1,136 @@
+const CELL_WIDTH = 101;
+const CELL_HEIGHT = 83;
+const COLUMNS = 5;
+const ROWS = 6;
+const FIELD_WIDTH = CELL_WIDTH * COLUMNS;
+const FIELD_HEIGHT = CELL_HEIGHT * ROWS;
+
+const PLAYER_START_POSITION_X = 202;
+const PLAYER_START_POSITION_Y = 385;
+const PLAYER_STEP_LENGTH_X = 101;

```

Comment:
Seems like we can "syc up" this with another constant.

Reply:
done